src/util.c

```diff
@@ -24,11 +24,6 @@
 #include "container.h"
 #include "../config.h"
 
-char *read_cmdline(void)
```

Comment:
What the function does is "return NULL", I'm not really removing anything useful that would need to be rewritten :)